src/components/AccountDetails/index.tsx

```diff
@@ -265,11 +265,12 @@ export default function AccountDetails({
               <AccountGroupingRow>
                 {formatConnectorName()}
                 <div>
-                  {connector !== injected && connector !== walletlink && (
+                  {connector !== injected && (
                     <WalletAction
                       style={{ fontSize: '.825rem', fontWeight: 400, marginRight: '8px' }}
                       onClick={() => {
-                        ;(connector as any).close()
+                        dispatch(setWalletOverride({ wallet: undefined }))
```

Comment:
i'm open to moving this somewhere more async, since i don't love that it's coupled. i've used redux-observable or redux-saga for this type of stuff before.

Reply:
we actually wouldn't need anything beyond basic redux now that i'm thinking more. eventually it'd be good to just pass an onClick into a generic WaleltModal component